docs/getting-started.md

```diff
@@ -8,7 +8,7 @@ If you encountered any issues above or have any queries, feel free to ask for he
 I would like you to try out a few of our keyboard shortcuts and send a couple message to help you get familiar with zulip terminal.
 
 #### What do you see?
-![An image of working Zulip Terminal](https://github.com/aditya-sarkar441/Screenshots/blob/master/Screenshot%202%202020-05-15%20at%209.31.03%20PM.png)
+<img src='https://user-images.githubusercontent.com/56690786/82113602-e27c0300-9774-11ea-9631-b5e6ee204a0c.png'>
```

Comment:
Is there any particular reason why you decided to go with HTML rather the existing markdown syntax?

Reply:
No such specific reasons. Both are working fine.

Reply:
For a cleaner diff, it would be better to replace the link in the existing markdown.

Reply:
Sure, I'll do it.